dev/com.ibm.ws.st.ui/src/com/ibm/ws/st/ui/internal/actions/ValidateConfigAction.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ */
+package com.ibm.ws.st.ui.internal.actions;
+
+import java.util.Iterator;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.viewers.ISelectionProvider;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.viewers.StructuredSelection;
+import org.eclipse.ui.actions.SelectionProviderAction;
+import org.eclipse.wst.validation.internal.ui.ValidationMenuAction;
```

Comment:
Could you create your own Validate menu action and the delegate just runs 
com.ibm.ws.st.core.internal.config.validation.ConfigurationValidator ?

Just concerned about the internal package of ValidationMenuAction.

Reply:
or use org.eclipse.wst.validation.ValidationFramework.validate?

Reply:
It has to be done this way.  Everything else was tried by Tim and this is the only thing that worked and still gave us the nice pop-up message with the validation results.  The problem is partly related to the resource type of server.xml in the Server's view.  We tried changing it but then it did not show well in the Server's view and that would have to be fixed in WTP.   Anyway - Tim, Elson and I had many discussions about this and this is the best way  to do it.  Also checked with Len and he said it should be safe to do this.